src/coprocessor/codec/mysql/duration.rs

```diff
@@ -457,6 +583,18 @@ mod tests {
             (b"00:00:00.777777", 2, Some("00:00:00.78")),
             (b"00:00:00.777777", 6, Some("00:00:00.777777")),
             (b"00:00:00.001", 3, Some("00:00:00.001")),
+            // NOTE:
```

Comment:
What's the note 🤣

Reply:
Sorry, I left the comments in my demos XD and forgot to paste in this file.